client/src/components/Models/index.vue

```diff
@@ -0,0 +1,184 @@
+<template>
+  <div>
+    <md-toolbar class="md-large">
+      <div class="md-toolbar-row">
+        <div class="md-toolbar-section-start">
+          <h3 class="md-title">Models</h3>
+        </div>
+
+        <div class="md-toolbar-section-end">
+          <!-- At this moment, restarting causes some inconsistencies between
+          the state held on the server and the state in the browser.
+          <md-button @click="handleRestartClick" class="md-accent">Restart</md-button>
+          -->
+          <md-button @click="handleNewModelClick" class="md-primary">New Model</md-button>
+        </div>
+      </div>
+      <md-list v-if="modelNames.length !== 0" class="model-list">
+        <Model
+          v-for="modelName in modelNames"
+          :key="modelName"
+          :name="modelName"
+          @delete-click="handleDeleteClick"
+          @edit-click="handleEditClick"
+          @copy-click="handleCopyClick"
+        />
+      </md-list>
+      <p v-else>No models created yet. Please click "New Model"</p>
+    </md-toolbar>
+    <div v-if="loadingModel"
+      style="display: grid; height: 100%">
+      <md-progress-spinner
+        :md-diameter="100"
+        :md-stroke="10"
+        md-mode="indeterminate"
+        style="margin: auto;"/>
+    </div>
+    <md-dialog :md-active.sync="showDialog">
+      <md-dialog-title>{{
+        currentOperation === OPERATIONS.create ?
+        'New Model' :
+        'Edit Model'
+      }}</md-dialog-title>
+      <Create
+        :params="currentModelParams"
+        @update-params="updateParams"
+        :model_metadata="currentModelMetaData"
+        @update-metadata="updateModelMetaData"
```

Comment:
That makes sense, this was mainly built to bubble state changes. So a response to this would be very similar to the comments about the state changes on the other ones too.

Reply:
This can be addressed in a different user story for the break if that is okay.